src/users/amoc_users_sup.erl

```diff
@@ -34,14 +33,17 @@
           sups_count :: pos_integer()
          }).
 
+-define(TABLE, amoc_users_sup_table).
+
 %% Supervisor
 
 %% @private
 -spec start_link() -> supervisor:startlink_ret().
 start_link() ->
     Ret = supervisor:start_link({local, ?MODULE}, ?MODULE, no_args),
     UserSups = supervisor:which_children(?MODULE),
-    IndexedSupsUnsorted = [ {Pid, N} || {{amoc_users_worker_sup, N}, Pid, _, _} <- UserSups],
+    IndexedSupsUnsorted = [ {Pid, N} || {{amoc_users_worker_sup, N}, Pid, _, _} <- UserSups,
+                                        is_integer(N), is_pid(Pid)],
```

Comment:
how is it possible that these filters could return false?

Reply:
That the first is an integer, not possible, but only we know that. That the second is not a pid can happen if a children is being restarted. Which is probably a catastrophe. So in all, this is kind of an artifact to make Eqwalizer happy that is is really strictly a list of {Pid,N} pairs.